### ISSUE TITLE
Quick Access does not allow rename 

### ISSUE THREAD

**Issue by robjansenitd** [microsoft/AzureStorageExplorer#1372]
When adding something to Quick Access it is not possible to rename it. Hence it is not possible to add the same item with the same name from e.g. different subscriptions.
I would like to be able to rename the item in Quick Access, without renaming the underlying item.

**Comment by JasonYeMSFT** [comment]
We now support adding different nodes with the same name to Quick Access. Would you still prefer a feature to allow you to customize your Quick Access nodes' display names after adding them or are you just looking for a way to distinguish nodes with the same name?

**Comment by okarika** [comment]
That would be extremely useful, currently there's no way to decide which item belongs to which subscription,resource group,etc. By adding a custom name to them we could quickly recognize them

**Comment by StefanGfK** [comment]
It would be extremely useful to rename the items in quick access.  This would not be renaming the blob/share itself, just giving it a distinguish name to know what it is.  Something like Project 1 (main) - where 'main' is the actual name

**Comment by kspdrgn** [comment]
Yes please, I just have 7 entries saying "$web" in my quick access.

**Comment by MRayermannMSFT** [comment]
Related: https://github.com/microsoft/AzureStorageExplorer/issues/2819
